node_doc.go

```diff
@@ -28,8 +29,11 @@ func (d NodeDoc) Order() uint64 {
 
 // Title of the document and computed with any ordering numbers and the
 // extension stripped off, usually for display purposes.
+// We normalize the title string to make sure all special characters
+// are represented in their composed form. For more on this topic see the
+// docblock of Node.Title().
 func (d NodeDoc) Title() string {
-	base := filepath.Base(d.path)
+	base := norm.NFC.String(filepath.Base(d.path))
```

Comment:
I like how this makes it symmetric to https://github.com/atelierdisko/dsk/pull/48/files#diff-f6846c341ab11ff34eeb7463ce2816ffR195